demos/network_agent/arch_config.yaml

```diff
@@ -19,6 +19,7 @@ system_prompt: |
 prompt_targets:
     - name: device_summary
       description: Retrieve network statistics for specific devices within a time range
+      http_method: POST
```

Comment:
change code to GET

Reply:
This isn't as simple as params need to be moved to path. Created issue  https://github.com/katanemo/archgw/issues/333